libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -197,8 +197,102 @@ const AP_Param::GroupInfo AP_Baro::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("BARO3_ID", 17, AP_Baro, sensors[2].bus_id, 0),
 #endif
-    
-    AP_GROUPEND
+
+    // @Param: COEF_XP
```

Comment:
we should break this into a separate param table with an enable param, maybe GND_WCOMP_EN ?